### ISSUE TITLE
Incorrect version on Serverless peer dependency?

### ISSUE THREAD

**Issue by jaikdean** [jacob-meacham/serverless-plugin-bind-deployment-id#8]
I am receiving the following error when trying to install the plugin:

```
$ npm install --save serverless-plugin-bind-deployment-id
redacted-project@1.0.0 /redacted/project/path
├── UNMET PEER DEPENDENCY serverless@>= 0.12.0
└─┬ serverless-plugin-bind-deployment-id@1.0.0 
  └── lodash@4.17.4 

npm WARN serverless-plugin-bind-deployment-id@1.0.0 requires a peer of serverless@>= 0.12.0 but none was installed.
```

Is the peer dependency version really meant to be `>= 1.12.0`, or is this an issue with Serverless being installed globally?

**Comment by jacob-meacham** [comment]
Hi @jaikdean the 1.0.0 version does require serverless 1.12+ because serverless made some changes to its lifecycle hooks that were incompatible with earlier versions of the plugin. If you're using an earlier version of serverless, you should use serverless-plugin-bind-deployment-id@0.1.0. Let me know if that works for you!

**Comment by jaikdean** [comment]
I thought that was the intention. There's a typo in the dependency (`0.12.0` instead of `1.12.0`).

I'm not hugely familiar with npm, but with Serverless 1.12.1 installed globally I received the error above. If I install Serverless locally within the project, the error goes away.

**Comment by jacob-meacham** [comment]
Right, that was my mistake! I've just published 1.0.1 with the correct Serverless dependency.

This is unfortunately how npm works - if the package is not installed locally, then npm considers the dependency unmet. I've opened #9 to explore whether or not serverless should be an explicit peer dependency.

Thanks!
